recipes/openssl/openssl_1.0.2g.oe

```diff
@@ -1,8 +1,5 @@
 require openssl.inc
 
-# Parallel make is simply broken for now.
-PARALLEL_MAKE = ""
```

Comment:
Ok.  So OpenEmbedded has a patch (taken from Gentoo) for parallel build races. In my book, that clearly wins over your "works on my build with -j8" ;)